fix_includes.py

```diff
@@ -1944,7 +2017,9 @@ def _GetSymbolNameFromForwardDeclareLine(line):
   """
   iwyu_namespace_re = re.compile(r'namespace ([^{]*) { ')
   symbolname_re = re.compile(r'([A-Za-z0-9_]+)')
-  namespaces_in_line = iwyu_namespace_re.findall(line)
+  # turn anonymous namespaces into their proper symbol representation
+  namespaces_in_line = iwyu_namespace_re.findall(line.replace(
```

Comment:
I'm curious how this is used, I hope there's a test to demonstrate :)

Reply:
@kimgr this was actually a bug in this method.  Given an anonymous namespace, the regular expression would fail to properly symbolicate the namespace, leading to global symbol.  Using nm and c++filt on a class in an anonymous namespace shows the symbol namespace as (anonymous namespace).   The test to check this behavior is below where I include an anonymous namespace inside of a regular namespace and are still able to get the proper namespace span and include new forward declares where needed.

Reply:
Hmm, would it be nicer to extract that bugfix out into a separate commit? It would be much easier to see the change that way, with a smaller, more focused diff. Or did it only manifest with nested namespaces?

Reply:
Also, I still don't see any mention of `(anonymous namespace)` elsewhere in this patch or in `fix_includes_test.py`, so I'm still not convinced the normalization is covered correctly.